recipes/tidy-html5/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```

```suggestion
include("conanbuildinfo.cmake")
```

To work with self._build_subfolder